airflow/operators/sql.py

```diff
@@ -120,16 +148,16 @@ def _convert_to_float_if_possible(s):
     return ret
 
 
-class SQLValueCheckOperator(BaseOperator):
+class SQLValueCheckOperator(BaseSQLOperator):
```

Comment:
Since this operator is in Airflow Core, can we release providers with the changes in this PR without corresponding core changes

Reply:
cc @potiuk

Reply:
I think so, the refactor on this operator is mainly to reduce duplicated code, and the logic, interface, and signature don't change in a backward incompatible one. But I don't know enough about how providers are released and how the core may affect it. @potiuk (sorry to ping you again..)

Reply:
I think it should work